nigsp/operations/timeseries.py

```diff
@@ -435,6 +435,14 @@ def graph_filter(timeseries, eigenvec, freq_idx, keys=["low", "high"]):
     return evec_split, ts_split
 
 
+def _trapezoid_compat(*args, **kwargs):
```

Comment:
Would it be possible to move it at the beginning of the file, or just before where it's needed, please? I know it does not change anything, but it's an old habit that is hard to die.

Reply:
Done